stdlib/public/core/Builtin.swift

```diff
@@ -768,6 +768,12 @@ func _trueAfterDiagnostics() -> Builtin.Int1 {
   return true._value
 }
 
+/// Whether client code is built with -D FOO_BAR, for testing only.
+@_alwaysEmitIntoClient
+private func _ifdefFooBar() -> Bool {
```

Comment:
Please don't add this function to the stdlib. You can create a test it by compiling a file with `-parse-stdlib` and explicitly `import Swift`.